easybuild/easyconfigs/0/3d-dna/3d-dna-180922-foss-2019a-Python-2.7.15.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'Tarball'
+
+name = '3d-dna'
+version = '180922'
+versionsuffix = '-Python-%(pyver)s'
+local_githash = '529ccf46599825b3047e58a69091d599e9858a19'
+
+homepage = 'https://github.com/theaidenlab/3d-dna'
+description = """3D de novo assembly (3D DNA) pipeline"""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = ['https://github.com/theaidenlab/%(name)s/archive']
+sources = ['%s.zip' % local_githash]
+checksums = ['348c3e019ea29e47382eb2d85228a56bc11b316c130afabae016ad8e7d7640ca']
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('LASTZ', '1.02.00'),
+    ('Java', '1.8', '', True),
```

Comment:
Please use `Java/11` (like we already do in other easyconfigs using a `2019a` toolchain)